assets/js/builder/controls/color.js

```diff
@@ -21,7 +21,7 @@ import { Palette } from './color/palette';
 
 		transparentColors: [ 'rgba(0, 0, 0, 0)', 'transparent' ],
 
-		colorClasses: [ 'color1-color', 'color2-color', 'color3-color', 'color4-color', 'color5-color' ],
+		colorClasses: [ 'color1-color', 'color2-color', 'color3-color', 'color4-color', 'color5-color', 'color-neutral-color' ],
```

Comment:
Any reason this is on a single line when the others are separated? Not a big deal, I am assuming is just based on the amount of values in the array.